README.md

```diff
@@ -80,6 +68,13 @@ Finally, setup the host environment by running:
 sgx-lkl-setup
 ```
 
+SGX-LKL works most performant with a Linux kernel that has support for userspace FSGSBASE instructions. Otherwise, support for thread local storage (TLS) must use emulated instructions, which reduces performance.
```

Comment:
Please can you put a line break per sentence?  It makes diffs much easier later.

Reply:
Personally, I think that this makes the underlying markdown files awkward to read. As long as paragraphs use no manual line wrapping, you can expect your editor (and git diff) to do the right thing.

Reply:
The GitHub UI doesn't do a great job with it (and the emails GitHub sends don't even try) and git diff doesn't on the command line for me.  GitX does a nice word-diff mode but most other git tooling doesn't seem to.  

Markdown intentionally requires a blank line between paragraphs specifically because of this.